### PR TITLE
Remove EL6 from the scripted installer, stop CircleCI builds

### PR DESCRIPTION
**Pull request by eorochena**
Removed EL6 from the scripted installer and stop EL6 builds in CIrcleCI, following https://github.com/StackStorm/discussions/issues/39

This PR doesn't cover `rpm` adjustments and `st2python` removal related to EL6 packaging.

### CODE REVIEW

File: .circle/packagecloud.sh [hunk 1]
```diff
@@ -37,8 +37,8 @@ function main() {
       fi
       ;;
     *)
-      echo $"Usage: deploy {trusty|xenial|bionic|el6|el7|el8} /tmp/st2-packages"
```
Comment (armab) [inline .circle/packagecloud.sh:40]: Please skip editing the `.circle/packagecloud.sh` file as it's just a generic script and may be used by the other repositories.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 Packages build environment is a *multi-container docker* application defined and managed with [docker-compose](https://github.com/docker/compose). It consists of four types of containers:
 
  - **Packaging runner** (https://quay.io/stackstorm/packagingrunner) - the main entry point, package build and test processing controller container.
- - **Packaging build** (https://hub.docker.com/r/stackstorm/packagingbuild/) - container where actual `.deb`/`.rpm` artifacts build takes place. It's used to bring up the build environment specific for OS distro. This means that different containers are available such as *packagingbuild:centos6*, *packagingbuild:xenial* correspondingly for CentOS 6 and Ubuntu Xenial.
+ - **Packaging build** (https://hub.docker.com/r/stackstorm/packagingbuild/) - container where actual `.deb`/`.rpm` artifacts build takes place. It's used to bring up the build environment specific for OS distro. This means that different containers are available such as *packagingbuild:centos7*, *packagingbuild:xenial* correspondingly for CentOS 6 and Ubuntu Xenial.
```
Comment (armab) [inline README.md:18]:
```suggestion
 - **Packaging build** (https://hub.docker.com/r/stackstorm/packagingbuild/) - container where actual `.deb`/`.rpm` artifacts build takes place. It's used to bring up the build environment specific for OS distro. This means that different containers are available such as *packagingbuild:centos7*, *packagingbuild:xenial* correspondingly for CentOS 7 and Ubuntu Xenial.
```

File: rake/build/setup.rake [hunk 3]
```diff
@@ -31,18 +31,17 @@ namespace :setup do
...
-      pipeline do
-        run hostname: opts[:testnode] do |opts|
-          repo_path = '/etc/yum.repos.d/stackstorm-el6-stable.repo'
-          execute :wget, "-nv https://bintray.com/stackstorm/el6/rpm -O #{repo_path}"
-          execute :sed, "-ir 's~stackstorm/el6~stackstorm/el6/stable~' #{repo_path}"
-          execute :yum, "--nogpgcheck -y install st2python"
-        end
-      end
-    end
+ task :install_st2_python do
```
Comment (armab) [inline rake/build/setup.rake:34]: Looks like the indentation for this code block was just broken by this change. Please revert it.

File: .circleci/config.yml [hunk 4]
```diff
@@ -17,7 +17,7 @@ jobs:
       - image: circleci/python:2.7
     working_directory: ~/st2-packages
     environment:
-      DISTROS: "xenial bionic el6 el7 el8"
```
Comment (armab) [inline .circleci/config.yml:20]: As we just removed `el6` distro, please also adjust the CircleCI job parallelism above from `5` -> `4`

**Comment by eorochena** [comment]
@armab I have implemented or the necessary changes to the PR also all CI checks have passed,.

Thanks for the suggestions.

**Pull request merged**